### DISCUSSION TITLE
yiwen-blog-website/docs/native/ReactNative/deugger

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#100] [Announcements]
# yiwen-blog-website/docs/native/ReactNative/deugger

背景

https://hao-yiwen.github.io/yiwen-blog-website/docs/native/ReactNative/deugger

**Comment by Hao-yiwen** [comment]
RN在hermes引擎中的调试真的是垃圾中的战斗机，关键是每次热更新就会删除所有断点信息

**Comment by Hao-yiwen** [comment]
使用flipper的hermes debug tools和debugger貌似能解决一些问题，需要验证。

**Comment by Hao-yiwen** [comment]
0.73.6中调试问题得到解决。
